src/components/ListItem.jsx

```diff
@@ -29,6 +54,8 @@ export function ListItem({ item, listId }) {
 				/>
 				{name}
 			</label>
+			<br />
+			<p className={indicatorClass}>{determineItemIndicator(item)}</p>
```

Comment:
Should this be part of the label? Maybe a `span` inside of the label?

Reply:
I think making it part of the label would be best, since clicking on the label will check the checkbox.   It all shows up on the same line as the checkbox because of CSS.